src/scanner/Scanner.ts

```diff
@@ -175,9 +175,11 @@ export class Scanner {
       }
 
       await git().silent(true).clone(cloneUrl.href, localPath);
+    } else if (localPath && serviceType !== ServiceType.local) {
```

Comment:
Change condition - don't check serviceType but check if it's git repo.

Reply:
I've moved the function to ScanningStrategyDetector.